src/functions_framework/__init__.py

```diff
@@ -219,6 +254,21 @@ def create_app(target=None, source=None, signature_type=None):
         # Add a dummy endpoint for GET /
         app.url_map.add(werkzeug.routing.Rule("/", endpoint="get", methods=["GET"]))
         app.view_functions["get"] = lambda: ""
+    elif signature_type == "cloudevent":
+        app.url_map.add(
+            werkzeug.routing.Rule(
+                "/", defaults={"path": ""}, endpoint=signature_type, methods=["POST"]
+            )
+        )
+        app.url_map.add(
+            werkzeug.routing.Rule(
+                "/<path:path>", endpoint=signature_type, methods=["POST"]
+            )
+        )
+
+        app.view_functions["cloudevent"] = _cloudevent_view_func_wrapper(
```

Comment:
```suggestion
        app.view_functions[signature_type] = _cloudevent_view_func_wrapper(
```

We either need to make this change or change what we're setting `endpoint` to when we're adding the rule to `url_map` above. Right now these just happen to be the same.

Furthermore, the pattern that the other two signature types follow is that the function is mounted under a `"run"` endpoint. I'm not sure it needs to be different here.